modules/cert-manager-istio-csr-uninstalling.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-integrating-istio.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cert-manager-istio-csr-uninstalling_{context}"]
+= Uninstalling the {cert-manager-operator} with istio-csr 
+
+
+To uninstall the {cert-manager-operator} with Istio-CSR, use the following steps.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have enabled the Istio-CSR.
+* You have created the `IstioCSR` custom resource.
+
+.Procedure
+
```

Comment:
Missing step one:

1. Remove the `IstioCSR` custom resource by running the following command:

```
$ oc -n <istio-csr_project_name> delete istiocsr.operator.openshift.io default
```